### PR TITLE
Adding custom password

### PR DESCRIPTION
**Pull request by ivannaranjo**
Implementation of the re-designed "Add Credential" dialog that allows manually adding the password.

This is how the dialog now looks like:
screen shot 2016-09-30 at 6 02 49 pm

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/ResetPassword/ResetPasswordViewModel.cs [hunk 1]
```diff
@@ -88,12 +137,25 @@ public ResetPasswordViewModel(ResetPasswordWindow owner, Instance instance, stri
...
-            OkCommand = new WeakCommand(OnOkCommand);
+            OkCommand = new WeakCommand(OnOkCommand, canExecuteCommand: false);
             CancelCommand = new WeakCommand(OnCancelCommand);
         }
 
         private async void OnOkCommand()
         {
+            if (ManualPassword)
+            {
+                Debug.WriteLine("The user is suplying the password.");
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/ResetPassword/ResetPasswordViewModel.cs]: nit: supplying

File: GoogleCloudExtension/GoogleCloudExtension/ResetPassword/ResetPasswordWindowContent.xaml [hunk 2]
```diff
@@ -19,7 +19,18 @@
             <TextBlock Margin="3,0,0,2.5" Text="{x:Static ext:Resources.ResetPasswordUserNameCaption}" />
             <TextBox Text="{Binding UserName, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"
                      IsEnabled="{Binding IsNotResettingPassword}"
-                     Margin="3,2.5" Height="30" x:Name="_passwordBox"/>
+                     Margin="3,2.5" Height="20" x:Name="_userName"/>
+            <RadioButton GroupName="password"
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/ResetPassword/ResetPasswordWindowContent.xaml]: Random note: This answers one of the questions I had...  I was doing something very wrong.

Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/ResetPassword/ResetPasswordWindowContent.xaml]: Awesome.

File: GoogleCloudExtension/GoogleCloudExtension/ResetPassword/ResetPasswordViewModel.cs (hunk 1 above)
Comment (ivannaranjo) [inline GoogleCloudExtension/GoogleCloudExtension/ResetPassword/ResetPasswordViewModel.cs]: Done.

**Pull request merged**